docs/reference/chisel-releases/chisel.yaml.md

```diff
@@ -0,0 +1,309 @@
+(chisel_yaml_ref)=
+# chisel.yaml
+
+The `chisel.yaml` file defines various configuration values for Chisel,
+for a given _chisel-release_.
+
+
+(chisel_yaml_location)=
+## Location
+
+The file must be placed in the root level of a {ref}`chisel-releases_ref`
+directory.
+
+
+(chisel_yaml_format_spec)=
+## Format specification
+
+(chisel_yaml_format_spec_format)=
+### `format`
+
+| Field    | Type  | Required | Supported values |
+| -------- | ----- | -------- | ---------------- |
+| `format` | `str` | Required | `v1`             |
+
+Used to define the supported schemas for the {ref}`chisel-releases_ref`.
+For example:
+
+```yaml
+format: v1
+```
+
+```{note}
+New formats are typically introduced with new _chisel-releases_ and may
+introduce disruptive changes to the previous formats.
+```
+
+
+(chisel_yaml_format_spec_archives)=
+### `archives`
+
+| Field      | Type              | Required |
+| ---------- | ----------------- | -------- |
+| `archives` | `dict(str, dict)` | Required |
+
+Tells Chisel which Ubuntu archives to fetch packages from.
+
+```{note}
+Chisel only supports fetching packages from the official Ubuntu archives,
+including ESM.
+```
+
+For example:
+
+```yaml
+archives:
+  ubuntu-esm-apps:
+    pro: esm-apps
+    priority: 16
+    version: 24.04
+    components: [main]
+    suites: [noble-apps-security, noble-apps-updates]
+    public-keys: [ubuntu-apps-key]
+```
+
+If {ref}`chisel_yaml_format_spec_archives_pro` is not specified, the archives
+point to:
+
+- http://archive.ubuntu.com/ for `amd64` and `i386` package architecture, and
+- http://ports.ubuntu.com/ubuntu-ports/ for other architectures.
+
+otherwise, the archive point to the Ubuntu Pro archives listed
+{ref}`below<chisel_yaml_format_spec_archives_pro>`.
+
+
+(chisel_yaml_format_spec_archives_default)=
+### `archives.<name>.default`
+
+| Field     | Type   | Required                                                                                            | Supported values |
+| --------- | ------ | --------------------------------------------------------------------------------------------------- | ---------------- |
+| `default` | `bool` | Required with multiple archives, if no {ref}`priorities<chisel_yaml_format_spec_archives_priority>` | `true`, `false`  |
+
+If `default` is `true`, Chisel fetches packages from this archive, unless
+otherwise specified by the field {ref}`"archive"<slice_definitions_format_archive>`
+in the slice definitions file.
+
+In case there are multiple archives, one, **and only one**, must be the default,
+**otherwise**, use {ref}`priorities<chisel_yaml_format_spec_archives_priority>`.
+
+```{tip}
+It is preferred to use {ref}`chisel_yaml_format_spec_archives_priority` instead
```

Comment:
We should say `default` is or will be deprecated, it is not a preference. In fact, default stops working the moment we introduce a priority which will be the case once we have `v2-archives`.

Reply:
Agreed. See 9bc3079